snowplow/iglu-client-embedded/schemas/com.metabase/structured_action/jsonschema/1-0-0

```diff
@@ -0,0 +1,60 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Generic event to track interactions and actions that happen within Metabase. This should be used as a default schema for tracking events. If more specific context is needed, we can use custom context entities (https://docs.snowplow.io/docs/understanding-your-pipeline/entities/).",
+    "self": {
+      "vendor": "com.metabase",
+      "name": "structured_action",
+      "format": "jsonschema",
+      "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+      "name": {
+        "description": "Name of the action. Noun (target) + Verb in the past (action) which define the action taken - e.g. question-created, dashboard-updated, dashboard-auto-apply-filter-enabled",
+        "type": "string",
+        "maxLength": 1024
+      },
+      "target_id": {
+        "description": "(Optional) ID of the entity that the action was performed on. E.g. the ID of the question that was created in a question_created event.",
+        "type": [
+            "string",
+            "null"
+        ],
+        "maxLength": 1024
+      },
+      "triggered_from": {
+        "description": "From where the action was taken. This can be generic like 'dashboard' or also more specific like 'dashboard_top_nav'.",
+        "type": "string",
+        "maxLength": 1024
+      },
+      "duration_ms": {
+        "description": "(Optional) Duration in milliseconds",
+        "type": [
+            "integer",
+            "null"
+        ],
+        "minimum": 0,
+        "maximum": 1800000
+      },
+      "result": {
+        "description": "(Optional) The outcome of the action (e.g. success, failure, …)",
+        "type": [
+            "string",
+            "null"
+        ],
+        "maxLength": 1024
+      },
+      "event_detail": {
+        "description": "(Optional) Can be used for additional details that describe the event, e.g. the type of question that was created in a question_created event.",
+        "type": [
+            "string",
+            "null"
+        ],
+        "maxLength": 1024
+      }
+    },
+    "required": [
+      "name", "triggered_from"
```

Comment:
@ranquild and me were discussing that we should potentially make the `triggered_from` also optional